apps/tb_meetings/src/tb_meetings_server.erl

```diff
@@ -0,0 +1,94 @@
+-module(tb_meetings_server).
+-behaviour(gen_server).
+-behaviour(deliverly_handler).
+-include_lib("deliverly/include/deliverly.hrl").
+-include_lib("deliverly/include/log.hrl").
+-define(SERVER, ?MODULE).
+
+%% API Function Exports
+-export([start_link/0]).
+
+%% gen_server Function Exports
+-export([init/1, handle_call/3, handle_cast/2, handle_info/2,
+         terminate/2, code_change/3]).
+
+%%% Deverly Handler callbacks.
+-export([authorize/2, handle_message/2, handle_client_message/2, client_disconnected/1]).
+
+-record(state, {
+  clients = #{} :: #{non_neg_integer() => de_client:client()}
+}).
+
+%% API Function Definitions
+
+start_link() ->
+  gen_server:start_link({local, ?SERVER}, ?MODULE, [], []).
+
+%% deliverly_handler Function Definitions
+
+authorize(Client, Data) ->
+  case proplists:get_value(<<"meeting_id">>, Data) of
+    undefined ->
+      {error, 3401};
+    Id ->
+      case deliverly_utils:auth_from_config(tb_meetings, Client, Data) of
+        true -> gen_server:call(?SERVER, {authorize, binary_to_integer(Id), Client});
+        false -> {error, 3401}
+      end
+  end.
+
+handle_message(Data, #{id := Id}) ->
+  gen_server:call(?SERVER, {handle_message, Id, Data});
+
+handle_message(_, _) -> ok.
+
+handle_client_message(_, _) -> ok.
+
+client_disconnected(Client) -> gen_server:call(?SERVER, {client_disconnected, Client}).
+
+%% gen_server Function Definitions
+
+init(_Args) ->
+  ?D(<<"Staring application: tb_meetings">>),
+  {ok, #state{}}.
+
+handle_call({authorize, Id, #de_client{} = Client}, _From, #state{clients = Clients} = State) ->
+  NewClient = Client#de_client{encoder = json_encoder, data = #{id => Id}},
+  NewClients = add_client(NewClient, Id, Clients),
+  {reply, {ok, NewClient}, State#state{clients = NewClients}};
+
+handle_call({client_disconnected, Client}, _From, #state{clients = Clients} = State) ->
```

Comment:
Line 60 is too long: handle_call({client_disconnected, Client}, _From, #state{clients = Clients} = State) ->.